requirements/optional.txt

```diff
@@ -1,2 +1,3 @@
-open3d
+open3d==0.9.0.0
+# The latest open3d need the support of file 'GLIBC_2.27', which only exist in Ubuntu 18.04, not in Ubuntu 16.04
```

Comment:
`The latest open3d need 'GLIBC_2.27' that only exists in Ubuntu 18.04, not in Ubuntu 16.04`